easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.0.6-intel-2016.02-GCC-4.9.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'BerkeleyGW'
+version = "1.0.6"
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+                 properties and the optical responses of a large variety of materials from bulk
+                 periodic crystals to nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2016.02-GCC-4.9'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']
+sources = ['BGW-%(version)s.tar.gz']
+checksums = ['8740562da41e87a62eb8d457675e4bfd']
+
+patches = ['BerkeleyGW_ignore_archmk.patch']
+
+files_to_copy = ['*']
+
+prebuildopts = 'COMPFLAG=-DINTEL PARAFLAG=-DMPI MATHFLAG=-DUSESCALAPACK FCPP="cpp -ansi" F90free="$MPIF90 -free" '
+prebuildopts += 'LINK="$MPIF90" FOPTS="$FFLAGS" MOD_OPT="-module " INCFLAG="-I" C_PARAFLAG=-DPARA CC_COMP="$MPICXX" '
```

Comment:
yes, it's weird, that's why I didn't commit :)

that is the one that we need to change less, since it's the only example that uses the intel compilers/mk/mpi stack. 

There is a `generic.mpi.linux.mk` file, but then we need to override almost all the other options again (well, at least we can skip `FCPP`, `REMOVE`, ...)